### PR TITLE
Jj switch

### PR DESCRIPTION
**Pull request by MartinaLof**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,27 @@
+<<<<<<< HEAD
```
Comment (fmavb) [inline .travis.yml]: Merge issues

File: README.md [hunk 2]
```diff
@@ -1,3 +1,46 @@
+<<<<<<< HEAD
```
Comment (fmavb) [inline README.md]: Merge issues

File: package.json [hunk 3]
```diff
@@ -1,3 +1,82 @@
+<<<<<<< HEAD
```
Comment (fmavb) [inline package.json]: Merge issues

File: .travis.yml (hunk 1 above)
Comment (ShayaP) [reply .travis.yml]: For these merge conflicts should I accept the incoming changes?

Comment (fmavb) [reply .travis.yml]: That's a good question. Let me take a look

Comment (ShayaP) [reply .travis.yml]: I looked at the files. I think the incoming changes have saucelabs added to them so they are probably more recent and we should accept those?

Comment (fmavb) [reply .travis.yml]: For all of the tooling files, you should accept the version which is currently on master

File: packages/core-hello/tests/core-hello_testcafe.js [hunk 4]
```diff
@@ -1,3 +1,24 @@
+<<<<<<< HEAD:test_testcafe/core-hello_testcafe.js
```
Comment (fmavb) [inline packages/core-hello/tests/core-hello_testcafe.js]: Merge conflict

File: packages/jj-input-number/tests/jj-input-number_test.js [hunk 5]
```diff
@@ -1,3 +1,124 @@
+<<<<<<< HEAD
```
Comment (fmavb) [inline packages/jj-input-number/tests/jj-input-number_test.js]: Merge conflict

File: packages/jj-input-number/tests/jj-input-number_testcafe.js [hunk 6]
```diff
@@ -1,3 +1,81 @@
+<<<<<<< HEAD:test_testcafe/jj-input-number_testcafe.js
```
Comment (fmavb) [inline packages/jj-input-number/tests/jj-input-number_testcafe.js]: Merge conflict

**Pull request merged**